flight/move_to_test.py

```diff
@@ -0,0 +1,94 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import goto
```

Comment:
Local imports should be relative to the root of the repostitory. Should be `import flight.goto`